ios/RCTCropView.m

```diff
@@ -8,17 +8,38 @@
 #import "RCTCropView.h"
 #import <TOCropViewController/TOCropView.h>
 #import <TOCropViewController/UIImage+CropRotate.h>
+#import <Photos/Photos.h>
 
 @implementation RCTCropView {
-    TOCropView * _inlineCropView;
+    TOCropView * _inlineCropView, * url;
```

Comment:
url should be an NSString instance. And by the looks of it, it can be a local var instead of an instance member.